test/adaptors/postgres-articles.test.ts

```diff
@@ -0,0 +1,144 @@
+import { literal, namedNode, quad } from '@rdfjs/data-model';
+import all from 'it-all';
+import 'jest-rdf';
+import { DatasetCore, NamedNode } from 'rdf-js';
+import pgPromise, { IBaseProtocol, IMain } from 'pg-promise';
+import PostgresArticles from '../../src/adaptors/postgres-articles';
+import ArticleNotFound from '../../src/errors/article-not-found';
+import NotAnArticle from '../../src/errors/not-an-article';
+import { schema } from '../../src/namespaces';
+import createArticle from '../create-article';
+import db from '../../src/db';
+import dataFactory from '../../src/data-factory';
+
+let pgp: IMain;
+let database: IBaseProtocol<IMain>;
+
+beforeAll(async (): Promise<void> => {
+  pgp = pgPromise();
+  database = pgp(db);
+  await PostgresArticles.createTables(database);
+});
+
+afterAll((): void => {
+  pgp.end();
+});
+
+describe('postgres articles #integration', (): void => {
+  beforeEach(async (): Promise<void> => {
+    await database.none('TRUNCATE articles');
```

Comment:
Other than setup this should have the same coverage as the `in-memory-articles.test.ts`: https://github.com/libero/article-store/blob/master/src/adaptors/in-memory-articles.ts

Reply:
Unique table names is another option, and would allow tests to be run in parallel.

Reply:
Similar to https://github.com/libero/content-api-bundle/blob/472c88159823a89bd71ed09534ab7234d7778b24/tests/Adapter/DoctrineItemsTest.php#L25-L57.

Reply:
> Unique table names is another option, and would allow tests to be run in parallel.

This would be a good refactor. Should we take it on here or as a followup?

Reply:
It's ok for the moment, the only tests using the database are in this file, which are run in sequence.